webapp/src/components/call_widget/call_widget.scss

```
@@ -0,0 +1,189 @@
+@keyframes callsKeyframesSlideFadeInOut {
```

Stylelint (17.14.1)

[error] Expected keyframe name "callsKeyframesSlideFadeInOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)